Makefile

```diff
@@ -47,6 +48,11 @@ zemu_install: tests_tools_build
 
 test_all:
 	make zemu_install
-	SUBSTRATE_PARSER_FULL=1 make
-	SUBSTRATE_PARSER_FULL=1 SUPPORT_SR25519=1 make buildS
+	make clean
+	APP_TESTING=1 SUBSTRATE_PARSER_FULL=1 PRODUCTION_BUILD=1 make buildX
```

Comment:
My bad, APP_TESTING was there for debugging purposes. I must have missed it when reviewing.

Reply:
And do we need a line for each device our can we just keep similar to what we did before and call
```
SUBSTRATE_PARSER_FULL=1 PRODUCTION_BUILD=1 make
SUBSTRATE_PARSER_FULL=1 SUPPORT_SR25519=1 PRODUCTION_BUILD=1 make buildS
```

Reply:
I found that the `SUPPORT_SR25519` flag is not properly propagated when first building nanoS without the flag and then building with it.
Inverting the lines was not possible because `make` does also execute `make clean`.

Reply:
I think if we do something like astar it might work :
```
test_all:
	make zemu_install
	SUBSTRATE_PARSER_FULL=1 make
	make clean_build
	SUBSTRATE_PARSER_FULL=1 SUPPORT_SR25519=1 make buildS
	make zemu_test
```
I tested, and it seems to me all test suites ran with success